clevertap-core/src/main/java/com/clevertap/android/sdk/ManifestInfo.java

```diff
@@ -330,4 +340,12 @@ private String _getManifestStringValueForKey(Bundle manifest, String name) {
             return null;
         }
     }
+
+    public String getProvider1() {
```

Comment:
@CTLalit - Manifest Label can be named 1 & 2, but can we have a better method name than `getProvider1` and `getProvider2`